File: dart/lib/model/v3_route_type.dart
part of swagger.api;

class V3RouteType {
  /* Name of transport mode */
  String routeTypeName = null;
/* Transport mode identifier */
  int routeType = null;

  V3RouteType();

  @override
  String toString() {
    return 'V3RouteType[routeTypeName=$routeTypeName, routeType=$routeType, ]';
  }

  V3RouteType.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    routeTypeName = json['route_type_name'];
    routeType = json['route_type'];
  }

  Map<String, dynamic> toJson() {
    return {
      'route_type_name': routeTypeName,
      'route_type': routeType
     };
  }

  static List<V3RouteType> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3RouteType>() : json.map((value) => new V3RouteType.fromJson(value)).toList();
  }

  static Map<String, V3RouteType> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3RouteType>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3RouteType.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3RouteType.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3RouteType {
    /// <summary>
    /// Name of transport mode
    /// </summary>
    /// <value>Name of transport mode</value>
    [DataMember(Name="route_type_name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "route_type_name")]
    public string RouteTypeName { get; set; }

    /// <summary>
    /// Transport mode identifier
    /// </summary>
    /// <value>Transport mode identifier</value>
    [DataMember(Name="route_type", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "route_type")]
    public int? RouteType { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3RouteType {\n");
      sb.Append("  RouteTypeName: ").Append(RouteTypeName).Append("\n");
      sb.Append("  RouteType: ").Append(RouteType).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
